src/xenia/kernel/xboxkrnl/xboxkrnl_module.cc

```diff
@@ -189,6 +190,11 @@ XboxkrnlModule::XboxkrnlModule(Emulator* emulator, KernelState* kernel_state)
                                        ordinals::XexExecutableModuleHandle,
                                        ppXexExecutableModuleHandle);
 
+  uint32_t ppExLoadedImageName =
+      memory_->SystemHeapAlloc(XFile::kMaxPathLength);
```

Comment:
And here

Reply:
Not sure if we need to always allocate all the 260 bytes here or just min(length, maxLength) + 1, by the way, the former is safer of course though.